workbench-core/authorization/src/dynamicAuthorization/dynamicAuthorizationService.test.ts

```diff
@@ -287,24 +288,74 @@ describe('DynamicAuthorizationService', () => {
   });
 
   describe('addUserToGroup', () => {
-    it('returns userID and groupID when user was successfully added to the group', async () => {
-      mockGroupManagementPlugin.addUserToGroup = jest.fn().mockResolvedValue({ data: { userId, groupId } });
+    let auditServiceWriteSpy: jest.SpyInstance;
+    let actor: object;
+    let source: object;
+    let action: string;
 
-      const { data } = await dynamicAuthzService.addUserToGroup({
-        groupId,
-        userId,
+    beforeAll(() => {
+      auditServiceWriteSpy = jest.spyOn(auditService, 'write');
+
+      action = 'addUserToGroup';
+      actor = mockUser;
+      source = {
+        serviceName: 'DynamicAuthorizationService'
+      };
+    });
+
+    afterEach(jest.resetAllMocks);
+
+    it('returns userID and groupID when user was successfully added to group', async () => {
+      const mockReturnValue = { data: { userId: 'userId', groupId: 'groupId' } };
+      mockGroupManagementPlugin.addUserToGroup = jest.fn().mockResolvedValue(mockReturnValue);
+
+      const requestBody = {
+        groupId: 'groupId',
+        userId: 'userId',
         authenticatedUser: mockUser
-      });
+      };
+
+      const { data } = await dynamicAuthzService.addUserToGroup(requestBody);
+
+      expect(mockGroupManagementPlugin.addUserToGroup).toBeCalledWith(requestBody);
```

Comment:
we've been using black box testing for the most part, so not sure if we want this test

Reply:
IMO since I'm mocking `addUserToGroup` return value test must know implementation details so it makes sense to test if it was invoked with proper values but I'm fine with removal if that's the agreement.